pkg/endpoint/policy.go

```diff
@@ -362,7 +362,7 @@ func (e *Endpoint) regenerateConsumable(owner Owner, labelsMap *identityPkg.Iden
 			rulesAdd, l4Rm, err = e.applyL4PolicyLocked(owner, labelsMap, e.L4Policy, c.L4Policy)
 			if err != nil {
 				// This should not happen, and we can't fail at this stage anyway.
-				e.getLogger().WithError(err).Fatal("L4 Policy application failed")
```

Comment:
Should you return here?

Reply:
+1 ^ Should be return here.

Reply:
And not enforce the policy at all?

Reply:
Wouldn't the policy enforced after this failure only be partially correct i.e some Label+L4 policy updates would have possibly failed.? Curious what's a better trade-off, to enforce partial policy or complain and not allow any policy.

Reply:
Sorry, overlooked something. Yes, we can still fail.